packages/tasks/src/tasks/zero-shot-object-detection/data.ts

```diff
@@ -0,0 +1,94 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "ImageNet (ILSVRC) 2012 is a widely used dataset of approximately 13 million labeled images organized by the WordNet hierarchy. It provides 1,281,167 training images, 50,000 validation images, and 100,000 test images across 1000 object classes."
+			id: "imagenet-1k"
+		}, 
+		{
+			description: "Microsoft COCO, a large-scale dataset of object instance segmentation, aims to advance object recognition by placing it in the context of scene understanding. It comprises 328k images with 2.5 million labeled instances of 91 common objects easily recognizable by a 4-year-old. ",
+			id: "merve/coco2017",
+		}
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "zero-sh-obj-detection_1.png"
```

Comment:
Can you upload this image by opening a PR to here? https://huggingface.co/datasets/huggingfacejs/tasks

Reply:
also would be nice to stick to general convention of how we name files (you can take a look at other task pages for example)